src/linalg.jl

```diff
@@ -80,7 +80,8 @@ function genperm(I::NTuple{N}, perm::NTuple{N}) where N
     ntuple(d-> (@inbounds return I[perm[d]]), Val(N))
 end
 
-function LinearAlgebra.permutedims!(dest::GPUArray, src::GPUArray, perm::NTuple{N, Integer}) where N
+function LinearAlgebra.permutedims!(dest::GPUArray, src::GPUArray, perm) where N
```

Comment:
The interface for `permutedims!` in Base is `permutedims!(dest, src::AbstractArray, perm)`. The original interface will dispatch the call to Base for vector perm unexpectedly. However, vector perm is a valid input for perm in GPUArrays.